### PR TITLE
feat: [VIC-309] replace email templates

### PR DESCRIPTION
**Pull request by amannn**
I've provided screenshots of the proposed changes here in the VIC-309 Jira ticket.

I've used a small script to replace dynamic values on the pages in the browser with something meaningful for the screenshots. <details>
<summary>Script</summary>

```js
[...document.querySelectorAll('[data-th-text]')].forEach(node => {
  const fieldName = node.getAttribute('data-th-text').replace(/[\$\{\}]/g,'');
  const value = {
    name_recipient: 'Susanne',
    name_sender: 'Julia',
    name_user: 'Susanne',
    name: 'Susanne',
    beratungsstelle: 'Happy Life',
  }[fieldName];
  node.textContent = value;
});
```
</details>

### CODE REVIEW

File: src/main/resources/templates/assign-enquiry-notification.html [hunk 1]
```diff
@@ -4,58 +4,46 @@
...
                      </div>
+                  </td>
+               </tr>
 
-                     <span style='clear: both;'><!-- clear --></span>
+               <tr>
+                  <td>
+                     <div style="color: #3f373f; margin: 16px; font-size: 16px; line-height: 21px; font-family: Nunito, 'Arial', 'sans-serif';">
+                        Liebe Grüße<br/>
+                        Ihr Team der Online Beratung<br/>
```
[the remark this answers is not in the document]
Comment (timomayer) [reply src/main/resources/templates/assign-enquiry-notification.html:45]: i would opt for it if effort is ok, feels more personal with a more specific name

[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.html:45]: Yes, that would be it. As just discussed, we'll do this in the second release.

File: src/main/resources/templates/assign-enquiry-notification.html [hunk 2]
```diff
@@ -4,58 +4,46 @@
...
-                  </div>
-               </td>
-               <td style='padding: 0; text-align: center; width: 90%; padding-right: 10%;'>
-                  <!-- logo -->
-
+            <thead style='background-color: #00008c;'>
+               <td style='padding: 16px 0; text-align: center; width: 100%;'>
                   <!-- claim -->
-                  <h1 style='color: #FFFFFF; font-size: 26px; font-weight: bold; line-height: 26px; padding-bottom: 0; margin-bottom: 0;'>Beratung & Hilfe</h1>
+                  <h1 style='color: #FFFFFF; font-size: 26px; font-weight: bold; line-height: 26px; padding-bottom: 0; margin: 0;'>Beratung & Hilfe</h1>
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.html:17]: Yes, thank you. This is also for the second release. I've added it to the ticket.

File: src/main/resources/templates/assign-enquiry-notification.html [hunk 3]
```diff
@@ -69,9 +57,9 @@ <h1 style='color: #FFFFFF; font-size: 26px; font-weight: bold; line-height: 26px
    <tr>
       <td style='text-align: center; padding: 0;'>
          <div style="text-align: center; padding-top: 8px;">
-            <a data-th-href="${urlimpressum}" style="color: #8C878C; text-decoration: none; font-size: 14px; line-height: 19px;">Impressum</a>
-            <span style="color: #8C878C; text-decoration: none; font-size: 14px; line-height: 19px;">&nbsp; | &nbsp;</span>
-            <a data-th-href="${urldatenschutz}" style="color: #8C878C; text-decoration: none; font-size: 14px; line-height: 19px;">Datenschutz</a>
+            <a data-th-href="${urlimpressum}" style="color: #666; text-decoration: none; font-size: 14px; line-height: 19px;">Impressum</a>
+            <span style="color: #666; text-decoration: none; font-size: 14px; line-height: 19px;">&nbsp; | &nbsp;</span>
+            <a data-th-href="${urldatenschutz}" style="color: #666; text-decoration: none; font-size: 14px; line-height: 19px;">Datenschutz</a>
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.html:62]: I've added the question also to the 2nd release ticket.

File: src/main/resources/templates/enquiry-notification-consultant.html [hunk 4]
```diff
@@ -4,62 +4,46 @@
...
-                  </td>
-               </tr>
-
                <tr>
                   <td cellspacing='0' cellpadding='0' style="padding: 0;" colspan="2">
-                    <div style="color: #3F373F; margin: 16px; font-size: 16px; line-height: 21px; font-family: 'Open Sans', 'OpenSans', 'Arial', 'sans-serif';">
+                    <div style="color: #3f373f; margin: 16px; font-size: 16px; line-height: 21px; font-family: Nunito, 'Arial', 'sans-serif';">
                      <b>Liebe(r) <span data-th-text="${name}"></span>,</b>
                      <br/><br/>
                        in der Beratungsstelle <span data-th-text="${beratungsstelle}"></span> ist eine neue Erstanfrage eingegangen.
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/enquiry-notification-consultant.html:28]: Yes, this is fine as all tenants will have an agency with a proper name :-)

File: src/main/resources/templates/assign-enquiry-notification.json [hunk 5]
```diff
@@ -1,29 +1,14 @@
...
 			"name_sender",
 			"name_user",
-			"url"
+			"url",
+			"urlimpressum",
+			"urldatenschutz"
 		]
-}
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.json:14]: Sorry, as discussed.

File: src/main/resources/templates/assign-enquiry-notification.json [hunk 6]
```diff
@@ -1,29 +1,14 @@
...
-		{
-			"filename":"illustration-top.png",
-			"filetype":"image/png"
-		},
-		{
-			"filename":"button_nachricht_ansehen_1x.png",
-			"filetype":"image/png"
-		}		
-	],
+	"templateImages" : [],
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.json:4]: yes, these are used by the mailService. Please change the url to `base.app.url` & `onlineberatung.net` to be generic in the 2nd release this functionality needs to be dynamized on tenant level.

File: src/main/resources/templates/assign-enquiry-notification.json [hunk 7]
```diff
@@ -1,29 +1,14 @@
...
 	"templateDataFields": 
 		[
 			"name_recipient",
 			"name_sender",
 			"name_user",
-			"url"
+			"url",
+			"urlimpressum",
+			"urldatenschutz"
```
[the remark this answers is not in the document]
Comment (koepferd) [reply src/main/resources/templates/assign-enquiry-notification.json:12]: I'll check this, thanks for the hint!

File: src/main/resources/templates/assign-enquiry-notification.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (amannn) [reply src/main/resources/templates/assign-enquiry-notification.html:45]: Thanks for chiming in! I talked to @koepferd in the afternoon and for the initial MVP release the plan is to not have anything tenant-specific in the email templates. That reduces the templates to a bare minimum unfortunately, so hopefully we can improve the situation here soon. We'll look into improvements in VIC-373. I hope that's ok for you!

**Review by koepferd — APPROVED** [review]

**Pull request merged**